script.js

```diff
@@ -324,6 +325,15 @@ hamburgerDiv.addEventListener('click', function () {
   }
 });
 
+if (params.get('dev') === 'true') {
+  document.addEventListener('click', function (event) {
```

Comment:
what If the user clicks on the navigation links, the navLinks.classList.remove('active'); line of code will be executed, which will remove the active class from all of the navigation links. This means that the user will not be able to tell which navigation link is currently active.

Reply:
For the dashboard site, navLinks refers to the class 'links' (hamburger menu) and class 'active' is added only to the class 'links'(hamburger menu), so it won't affect any other navbar links when 'active' gets removed, also there are no links added for the options in hamburger menu, so clicking on any of them opens dashboard site.

Reply:
> For the dashboard site, navLinks refers to the class 'links' (hamburger menu) and class 'active' is added only to the class 'links'(hamburger menu), so it won't affect any other navbar links when 'active' gets removed, also there are no links added for the options in hamburger menu, so clicking on any of them opens dashboard site.

cool, what about below code

document.addEventListener('click', function (event) {
  if (!navActive.contains(event.target) && !event.target.closest('.nav-link')) {
    navLinks.classList.remove('active');
    toggle = true;
  }
});

Reply:
!navActive.contains(event.target)
-here the constant 'navActive'  contains the class '.nav-links', so it will automatically check if the clicked element is a descendent of '.nav-links' or not.